algorithms/src/Kokkos_Random.hpp

```diff
@@ -915,13 +915,14 @@ class Random_XorShift64_Pool {
     state_ = state_data_type("Kokkos::Random_XorShift64::state", num_states_,
                              padding_);
 
-    typename state_data_type::HostMirror h_state =
+    typename state_data_type::host_mirror_type h_state =
```

Comment:
I would do a separate PR for just this change (*i.e.* search replace of `HostMirror` with `host_mirror_type`.

Reply:
Will do, thanks for looking into this.